### DISCUSSION TITLE
option placeholder

### DISCUSSION THREAD

**Discussion by mincheol-jeong** [Budibase/budibase#10010] [Help]
If you enter it in the option placeholder part, only Choose an option keeps floating and does not change. Is it like that? It was possible in version 2.3.8.

**Comment by melohagan** [comment]
Hey @mincheol-jeong 

Could you provide a bit more detail and a screenshot of the issue you're facing please?

**Reply by mincheol-jeong** [reply]
My problem is the same as the issue. When will the merge  happen? #9993

**Comment by melohagan** [comment]
Thanks for linking the issue @mincheol-jeong 

I have approved the PR for the fix, and it will likely be merged in for the next release at the latest.
